spec/u-namespace.md

```diff
@@ -34,11 +34,11 @@ or any other structured formatted results.
 
 Ignored when formatting a message to a string.
 
-The value of the `u:id` _option_ MUST be a _literal_ or a
+The `u:id` _option value_ MUST be a _literal_ or a
 _variable_ whose _resolved value_ is either a string
 or can be resolved to a string without error.
 For other values, a _Bad Option_ error is emitted
-and the `u:id` option is ignored.
+and the `u:id` _option_ and its _option value_ are ignored.
```

Comment:
There's no need to add option value here, is there?

```suggestion
and the `u:id` _option_ is ignored.
```

Reply:
I was thinking that explicitly ignoring the _option value_ would allow for an implementation to not emit any further errors that might arise during its resolution.

Similar phrasing is used further down in this section, so if you think that we ought to change this, we ought to change all of them.

Reply:
That seems appropriately cautious.